oe-init/internal/gen-oe-setup.py

```diff
@@ -7,13 +7,14 @@ def main():
 	parser = argparse.ArgumentParser(description='Generate an OE setup script')
 
 	parser.add_argument('--base-path', help='Path to base working directory', default=None)
-	parser.add_argument('--layers-path', help='Path to OE layers directory', default='openembedded'
-)
+	parser.add_argument('--layers-path', help='Path to OE layers directory', default='openembedded')
+
```

Comment:
drop this new line

Reply:
There's no new line here, I just moved the round bracket back up to line #10